packages/server/graphql/mutations/helpers/updateQualAIMeetingsCount.ts

```diff
@@ -0,0 +1,23 @@
+import updateTeamByTeamId from '../../../postgres/queries/updateTeamByTeamId'
+import {DataLoaderWorker} from '../../graphql'
+
+const updateQualAIMeetingsCount = async (
+  meetingId: string,
+  teamId: string,
+  dataLoader: DataLoaderWorker
+) => {
+  const [meetingMembers, team, reflections, meeting] = await Promise.all([
+    dataLoader.get('meetingMembersByMeetingId').load(meetingId),
+    dataLoader.get('teams').load(teamId),
+    dataLoader.get('retroReflectionsByMeetingId').load(meetingId),
+    dataLoader.get('newMeetings').load(meetingId)
+  ])
+  if (meetingMembers.length < 3 || !team || !meeting.summary || reflections.length < 5) return
+  const {qualAIMeetingsCount} = team
+  const updates = {
+    qualAIMeetingsCount: qualAIMeetingsCount + 1
+  }
+  await updateTeamByTeamId(updates, teamId)
```

Comment:
-1 since we load team by dataloader we should update or invalidate the cache here for it. We should do this in all cases where we aren't sure if the object will be used subsequently. Since this is a helper we don't know anything about the calling site.

Reply:
Makes sense - updated!